import { App } from "./app";

const app = new App();
app.run();

// Exception Error handling
process.on("uncaughtException", (err: Error) => {
  console.log("UNCAUGHT EXCEPTION! ðŸ’¥ Shutting down...");
  console.log(err.name, err.message);
  process.exit(1);
});

process.on("unhandledRejection", (err: Error | any) => {
  console.log("UNHANDLED REJECTION! ðŸ’¥ Shutting down...");
  console.log(err.name, err.message);
  app.server.close();
});
